test/ProfilerTest.cpp

```diff
@@ -36,14 +133,56 @@ TEST(ProfilerTest, CreateForFile) {
 }
 
 TEST(ProfilerTest, CreateForUDP) {
-  {
+
```

Comment:
We had this HumanPrinter stuf copy-pasted a lot... I am afraid it will not be maintainable... although it is not as important. now.